src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -186,6 +190,12 @@ public Mono<Void> replaceFilterWithScript(UUID id, String userId) {
 
     public Mono<Void> deleteElement(UUID id, String userId) {
         return directoryService.deleteElement(id, userId)
-            .doOnSuccess(e -> directoryService.deleteDirectoryElement(id, userId).subscribe());
+                .doOnSuccess(e -> directoryService.deleteDirectoryElement(id, userId).subscribe())
+                // FIXME dirty fix to ignore errors and still delete the elements in the directory-server. To delete when handled properly.
```

Comment:
Why is it a fix ?

Reply:
because now if an exception is thrown we still procced to delete the element in the directory

Reply:
I think it's not a fix 
it's the desired  behavior